build/configs/artik053/audio/defconfig

```diff
@@ -815,7 +815,7 @@ CONFIG_AUDIO_BUFSIZE=2048
 #
 # CONFIG_AUDIO_FORMAT_AC3 is not set
 # CONFIG_AUDIO_FORMAT_DTS is not set
-CONFIG_AUDIO_FORMAT_PCM=y
+# CONFIG_AUDIO_FORMAT_PCM is not set
```

Comment:
please check these configs are needed or not.

Reply:
I will remove CONFIG_AUDIO_FORMAT_PCM. But I will keep the other FORMAT_xxx configs. They might be required in future if h/w supports those formats.